### PR TITLE
[CHEC-1358] Update products, cart, checkout to use new variants

### PR DESCRIPTION
**Pull request by jaepass**
- Updated product detail page, cart items, checkout summary and order to use new variants API
- Merchant store updated to latest API version
- Minor a11y fixes

Tested and all good

### CODE REVIEW

**Comment by ScopeyNZ** [comment]
Can you update the version specified to the SDK? It should be specifically set if it isn't already

**Comment by jaepass** [comment]
Just update the SDK to latest version?

**Comment by ScopeyNZ** [comment]
No I mean we'll need to specific the `Chec-Version` header when creating Commerce.js - otherwise this update will break for people who have their API version set in the dashboard to an older version

File: components/checkout/Confirm.js [hunk 1]
```diff
@@ -35,9 +35,8 @@ class Confirm extends Component {
       <button
         onClick={this.handlePrint}
         className="d-flex align-items-center text-decoration-underline cursor-pointer mt-3 mt-sm-0 no-print bg-transparent"
-        role="button"
       >
-        <img src="/icon/print.svg" className="mr-2 w-20 no-print"/>
+        <img src="/icon/print.svg" className="mr-2 w-20 no-print" title="Printer icon" alt=""/>
```
Comment (robbieaverill) [inline components/checkout/Confirm.js:39]: I think having an empty alt tag here is enough. The button that contains this image already has "print receipt" for screenreaders, the empty alt tag is enough for screenreaders to ignore this image.

Same comment for the other icon change below as they're both [decorative images](https://www.w3.org/WAI/tutorials/images/decorative/).

**Review by robbieaverill — COMMENTED** [review]
LGTM, I just had a small suggestion for your icon image changes

**Pull request merged**